src/app/tests/suites/certification/Test_TC_S_2_5.yaml

```diff
@@ -106,32 +148,137 @@ tests:
       verification: |
           ./chip-tool scenesmanagement remove-all-scenes 0x0001 1 1
 
-          CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0005 Command 0x0000_0003
-          CHIP:TOO:   RemoveAllScenesResponse: {
-          CHIP:TOO:     status: 1
-          CHIP:TOO:     groupID: 1
-          CHIP:TOO:    }
+          Verify the RemoveAllScenesResponse with following fields:
+          Status is SUCCESS
+          Group ID is 0x0001  on the TH(Chip-tool) log and below is the sample log provided for the raspi platform:
+
+          [1706764352.114217][4432:4434] CHIP:DMG: },
+          [1706764352.114233][4432:4434] CHIP:DMG: Received Command Response Data, Endpoint=1 Cluster=0x0000_0062 Command=0x0000_0003
+          [1706764352.114241][4432:4434] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0062 Command 0x0000_0003
+          [1706764352.114257][4432:4434] CHIP:TOO:   RemoveAllScenesResponse: {
+          [1706764352.114264][4432:4434] CHIP:TOO:     status: 0
+          [1706764352.114267][4432:4434] CHIP:TOO:     groupID: 1
+          [1706764352.114269][4432:4434] CHIP:TOO:    }
       disabled: true
 
     - label: "Step 4a: TH reads from the DUT the SceneTableSize attribute"
       verification: |
           ./chip-tool scenesmanagement read scene-table-size 1 1
 
-          [1688019474.695136][1726:1728] CHIP:DMG: }
-          [1688019474.695335][1726:1728] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0005 Attribute 0x0000_0006 DataVersion: 1550229741
-          [1688019474.695397][1726:1728] CHIP:TOO:   SceneTableSize: (Default table size: 16)
+          Verify the "SceneTableSize" attribute value is 16  on the TH (Chip-tool) and below is the sample log provided for the raspi platform:
```

Comment:
Is this supposed to always be 16? Or is this supposed to be noted as `SceneTableSize` for a future value

Reply:
`SceneTableSize`, will correct.